include/global.h

```diff
@@ -11,6 +11,7 @@
 #define JUT_ASSERT(...)
 #define ASSERT(...)
 #define LOGF(FMT, ...)
+#define FLAG_HAS(V, F) (((V) & (F)) == 0)
```

Comment:
```suggestion
#define FLAG_ON(V, F) (((V) & (F)) == 0)
```
might be more verbose